api_core/.coveragerc

```diff
@@ -0,0 +1,14 @@
+[run]
+branch = True
+
+[report]
+fail_under = 100
+show_missing = True
+exclude_lines =
+    # Re-enable the standard pragma
+    pragma: NO COVER
+    # Ignore debug-only repr
+    def __repr__
+    # Ignore abstract methods
+    raise NotImplementedError
+    raise NotImplementedError()
```

Comment:
This is line redundant, is it not? Otherwise line 11 would not work.

Reply:
So it is, removed.